velox/core/PlanNode.cpp

```diff
@@ -433,12 +433,15 @@ void PlanNode::toString(
         std::stringstream& stream)> addContext) const {
   const std::string indentation(indentationSize, ' ');
 
-  stream << indentation << "-> " << name();
+  stream << indentation << "-- " << name();
 
   if (detailed) {
     stream << "[";
     addDetails(stream);
     stream << "]";
+    stream << " -> ";
+    const auto& outputType = this->outputType();
```

Comment:
nit: any particular reason for this temp variable? Why not simply `outputType()->printChildren(stream)`?